jscomp/test/js_date.js

```diff
@@ -1,5 +0,0 @@
-'use strict';
-
```

Comment:
can we keep this file to rename it as `oo_js_test_date.ml`, it still serves the test functionality